packages/fxa-payments-server/package.json

```diff
@@ -135,10 +135,10 @@
     ],
     "coverageThreshold": {
       "global": {
-        "branches": 40,
-        "functions": 40,
-        "lines": 50,
-        "statements": 50
+        "branches": 50,
```

Comment:
Slight increase in percentages for `coverageThreshold` - doesn't seem like much, but some directories are 100% now